exes/ethercat/systemd/ethercat@.service

```diff
@@ -4,7 +4,7 @@ Requires=dbus.socket
 After=dbus.socket
 
 [Service]
-ExecStart=/usr/bin/ethercat --iface %i
+ExecStart=/usr/local/bin/ethercat --iface %i
```

Comment:
Can you template this on CMAKE_INSTALL_BINDIR so it does not break again?

Reply:
Will do, the variable `CMAKE_INSTALL_PREFIX` is already used in another place in the repo so using that variable.